clients/cobol-lsp-vscode-extension/src/test/suite/lsp.spec.test.ts

```diff
@@ -590,28 +590,37 @@ suite("Integration Test Suite", function () {
     .timeout(helper.TEST_TIMEOUT)
     .slow(1000);
 
-  test("Show transition copybook errors", async () => {
-    // Open program with error inside a copybook
-    await helper.showDocument("TESTCPY2.cbl");
-    const progUri = await helper.getUri("TESTCPY2.cbl");
+  test(
+    "Show transition copybook errors" +
+      "TC288744: Underscore a copy statement if its copybook contains error: nested copybooks" +
+      'TC247996: Nested copybooks with "no extension" are supported',
```

Comment:
Is this a valid scenario ? I am under the impression that we need to have a setting with empty string for the copybook extension settings, this to work.